docs/_ext/github_links.py

```diff
@@ -0,0 +1,136 @@
+import ast
+import functools
+import importlib.util
+import pathlib
+
+BASE_PATH = pathlib.Path(__file__).parents[2]
```

Comment:
Can we deffer accessing `__file__` or don't use it here? (check out ticket-32316 and 9ee693bd6cf4074f04ec51c6f3cfe87cad392f12)

Reply:
Wow I had no idea. This seems like it should be part of the automated testing/linting steps rather than being manually checked, so that we avoid back and forth. Should we create a ticket for doing that?

Reply:
> Should we create a ticket for doing that?

This could be a part of ticket-30950. For now, we're trying to make life less painful for folks with frozen environments.

Reply:
Yeah, I got it. I'll try to do it soon.